CONTRIBUTING.md

```diff
@@ -56,19 +72,54 @@ python -m tox -e windows-dev # (Windows)
 
 This will
 
-1. Create a virtual environment located at `.tox/dev`.
-2. Install all the dependencies for PyBaMM, including the ones for documentation and development.
-3. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+a. Create a virtual environment located at `.tox/dev`.
+
+b. Install all the dependencies for PyBaMM, including the ones for documentation and development.
 
-Finally, activate your environment.
+c. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+
+5. Finally, activate your environment.
 
 
 ```bash
 source .tox/dev/bin/activate # (GNU/Linux and MacOS)
 #
-.tox\dev\Scripts\activate.bat # (Windows)
+.tox\windows-dev\Scripts\activate.bat # (Windows)
+```
+6. Inside the virtual environment, type:
+```bash
+pip install tox # (Skip this step if you used pipx above)
+```
+
+### 2. Without using tox
+1. Install virtual environment using the command below:
+```bash
+pip install virtualenv
+```
+2. Create a virtual environment `env` within your current directory using the command below:
+```bash
+python -m venv env 
+# (You can then “activate” the environment using the command below)
+env\Scripts\activate.bat
+# (This whole step can be ignored if you don't want a virtual environment)
 ```
 
+3. Installing PyBaMM
+
+This can be acieved by forking the repository and cloning it in your machine.
+
+Or type:
+```bash
+pip install pybamm
+```
+
+Additionally you can install sundials on mac using:
+```bash
+brew install sundials
```

Comment:
Sundials should be installed _before_ installing PyBaMM.
See comment above

Reply:
Okay, I'll make the changes, thanks!